### ISSUE TITLE
webpack task caching ignores changes to custom webpack config files

### ISSUE THREAD

**Issue by japgolly** [scalacenter/scalajs-bundler#112]
Say you have

```scala
webpackConfigFile in fastOptJS := Some(baseDirectory.value / "webpack-dev.js"),
webpackConfigFile in fullOptJS := Some(baseDirectory.value / "webpack-prod.js"),
```

Then:
1. `fastOptJS::webpack`
2. Modify `webpack-dev.js`.
3. `fastOptJS::webpack` ← should re-run webpack but doesn't.

**Comment by danxmoran** [comment]
@japgolly / @julienrf is this still a problem in 0.7? I've been trying to reproduce in a minimal scripted test without any luck.

**Comment by julienrf** [comment]
I think this has been fixed by 32d852d533f8832f78d5152425ad97453fb40e83
